src/main/cpp/option_processor.cc

```diff
@@ -151,6 +151,11 @@ std::unique_ptr<CommandLine> OptionProcessor::SplitCommandLine(
         std::move(path_to_binary), std::move(startup_args), "", {}));
   }
   string& command = args[i];
+  // Distinguish an empty command from the case of no command above.
+  if (command.empty()) {
+    blaze_util::StringPrintf(error, "Command cannot be the empty string (try 'help')");
```

Comment:
nit: mind making this symmetric with the "no command" case and say "try 'bazel help'" instead?